tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -200,6 +200,80 @@ def test_positive_create_2(self, test_data):
         )
 
     @data(
+        {'label': gen_string("alpha", 15)},
+        {'label': gen_string("alphanumeric", 15)},
+        {'label': gen_string("numeric", 15)},
+    )
+    def test_create_lifecycle_environment_by_label(self, test_data):
+        """@Test: Create lifecycle environment with valid name and label
+
+        @Feature: Lifecycle Environment
+
+        @Assert: Lifecycle environment with label is created
+
+        """
+        payload = {
+            'organization-id': self.org['id'],
+            'name': test_data['label'],
+            'label': test_data['label'],
+        }
+
+        try:
+            new_le = make_lifecycle_environment(payload)
```

Comment:
Try to avoid single-use variables. Something like this is better:

```python
make_lifecycle_environment({
    'organization-id': self.org['id'],
    'name': test_data['label'],
    'label': test_data['label'],
})
```